docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ReplaceMissingValuesColumnOptions.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.ML.Data;
+using Microsoft.ML.SamplesUtils;
+using Microsoft.ML.Transforms;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class ReplaceMissingValuesColumnOptions
```

Comment:
>ReplaceMissingValuesColumnOptions [](start = 24, length = 33)

Please rename file and class to ReplaceMissingValuesWithOptions to match other samples (although this particular one doesn't sound ideal)